cmd/experimental/kjobctl/pkg/cmd/create/create.go

```diff
@@ -320,6 +334,10 @@ var createModeSubcommands = map[string]modeSubcommand{
 				"MinReplicas denotes the minimum number of desired Pods for this worker group.")
 			subcmd.Flags().StringToIntVar(&o.MaxReplicas, maxReplicasFlagName, nil,
 				"MaxReplicas denotes the maximum number of desired Pods for this worker group, and the default value is maxInt32.")
+			subcmd.Flags().StringVar(&o.Priority, priorityFlagName, "",
+				"Apply priority for the entire workload.")
+			subcmd.Flags().BoolVar(&o.SkipPriorityValidation, skipPriorityValidationFlagName, false,
+				"Skip workload priority class validation. Add priority class label even if the class does not exist.")
 		},
 	},
 	"slurm": {
```

Comment:
Also for about Slurm.

[--priority NAME] is not a Slurm flag we should move it to flags before dashes.